tests/core/pyspec/eth2spec/test/electra/epoch_processing/test_process_pending_consolidations.py

```diff
@@ -4,7 +4,7 @@
 )
 from eth2spec.test.context import (
     spec_state_test,
-    with_electra_and_later,
+    with_electra_and_later
```

Comment:
```suggestion
    with_electra_and_later,
```
usual thing in python to leave commas at the end, should we add one more line this trick would reduce the size of the diff